EnglishFilter/sections/antiadblock.txt

```diff
@@ -681,6 +681,8 @@ smallseotools.com#@#.adsbygoogle
 !
 !
 ! anti-adb
+! https://github.com/AdguardTeam/AdguardFilters/issues/118061
+||bagi.co.in/*/js/check.js
```

Comment:
Wrong place - between existing comment+rule.

Reply:
@gork7777

Reply:
@Alex-302 Which row should it be in?